LiskHQ/lisk-hub
# Add followed accounts to account profile page - Closes #907

### What was the problem?
There was no way of following an account via their profile page

### How did I fix it?
Implemented it

### How to test it?
- Follow an account
- Check dashboard
- Go back unfollow
- Try variations of follow/unfollow/changing title etc

### Review checklist
- The PR solves #907
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
